src/overlays/actors/ovl_En_Bal/z_en_bal.c

```diff
@@ -64,108 +135,1023 @@ static DamageTable D_80A63CE0 = {
     /* Powder Keg     */ DMG_ENTRY(0, 0x0),
 };
 
-// sColChkInfoInit
-static CollisionCheckInfoInit2 D_80A63D00 = { 1, 0, 0, 0, MASS_IMMOVABLE };
+static CollisionCheckInfoInit2 sColChkInfoInit = { 1, 0, 0, 0, MASS_IMMOVABLE };
 
-// static ColliderJntSphElementInit sJntSphElementsInit[1] = {
-static ColliderJntSphElementInit D_80A63D0C[1] = {
+static ColliderJntSphElementInit sJntSphElementsInit[1] = {
     {
-        { ELEMTYPE_UNK0, { 0xF7CFFFFF, 0x00, 0x00 }, { 0xF7CFFFFF, 0x00, 0x00 }, TOUCH_NONE | TOUCH_SFX_NORMAL, BUMP_ON, OCELEM_ON, },
-        { 6, { { 2400, 0, 0 }, 50 }, 100 },
+        {
+            ELEMTYPE_UNK0,
+            { 0xF7CFFFFF, 0x00, 0x00 },
+            { 0xF7CFFFFF, 0x00, 0x00 },
+            TOUCH_NONE | TOUCH_SFX_NORMAL,
+            BUMP_ON,
+            OCELEM_ON,
+        },
+        { TINGLE_LIMB_BALLOON, { { 2400, 0, 0 }, 50 }, 100 },
     },
 };
 
-// static ColliderJntSphInit sJntSphInit = {
-static ColliderJntSphInit D_80A63D30 = {
-    { COLTYPE_NONE, AT_ON | AT_TYPE_ENEMY, AC_ON | AC_TYPE_PLAYER, OC1_ON | OC1_TYPE_ALL, OC2_TYPE_1, COLSHAPE_JNTSPH, },
-    ARRAY_COUNT(sJntSphElementsInit), D_80A63D0C, // sJntSphElementsInit,
+static ColliderJntSphInit sJntSphInit = {
+    {
+        COLTYPE_NONE,
+        AT_ON | AT_TYPE_ENEMY,
+        AC_ON | AC_TYPE_PLAYER,
+        OC1_ON | OC1_TYPE_ALL,
+        OC2_TYPE_1,
+        COLSHAPE_JNTSPH,
+    },
+    ARRAY_COUNT(sJntSphElementsInit),
+    sJntSphElementsInit,
+};
+
+static s16 sBuyMapOptions[TINGLE_MAP_MAX][2] = {
+    { TINGLE_MAP_CLOCK_TOWN, TINGLE_MAP_WOODFALL },   { TINGLE_MAP_WOODFALL, TINGLE_MAP_SNOWHEAD },
```

Comment:
```suggestion
    { TINGLE_MAP_CLOCK_TOWN, TINGLE_MAP_WOODFALL }, // TINGLE_MAP_CLOCK_TOWN
    { TINGLE_MAP_WOODFALL, TINGLE_MAP_SNOWHEAD }, // TINGLE_MAP_WOODFALL
```
Could you add comments just to highlight that `mapId` is both the index and the values of `sBuyMapOptions`